deploy/katello.spec

```diff
@@ -116,8 +116,9 @@ Summary:    Common runtime components of %{name}
 Requires:       wget
 Requires:       curl
 Requires:       %{?scl_prefix}rubygem-katello
-Requires:       rubygem-hammer_cli 
+Requires:       rubygem-hammer_cli
 Requires:       rubygem-hammer_cli_foreman
+Requires:       rubygem-hammer_cli_foreman_docker
```

Comment:
If the katello cli requires it then this is unneeded.
On Apr 20, 2015 5:50 PM, "thomasmckay" <notifications@github.com> wrote:

> In deploy/katello.spec
> <https://github.com/Katello/katello/pull/5187#discussion_r28733136>:
>
> >  Requires:       rubygem-hammer_cli_foreman
> > +Requires:       rubygem-hammer_cli_foreman_docker
>
> Unfortunately, I think hammer-cli-katello now relies on
> hammer-cli-foreman-docker. Certainly the docker plugin should not be
> shipped w/ SAM, but the cli could be and then disabled by SAM install.
>
>
> https://github.com/Katello/hammer-cli-katello/blob/master/hammer_cli_katello.gemspec#L42
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/katello/pull/5187/files#r28733136>.
>